packages/new-deploy/scripts/deploy.ts

```diff
@@ -0,0 +1,34 @@
+import DeployContext from '../src/DeployContext';
+import * as config from '@nomad-xyz/configuration';
+import * as ethers from 'ethers';
+import { NonceManager } from '@ethersproject/experimental';
+import fs from "fs";
+
+run();
+
+/*
+* get published config
+* modify to add a network
+* run the deploy script
+* re-publish config
+* */
+const config = {}; // TODO: get published (or local??) config
```

Comment:
idea state:
- script accepts a string as input from `process.argv` or env var
- string is tried as a filepath, then a URL

Reply:
- implemented local filepath
- added ticket for attempting URL: https://github.com/nomad-xyz/monorepo/issues/112